tests/drivers/test_elasticsearch.py

```diff
@@ -344,6 +344,58 @@ def test_fetch_points_updates_read_on(self):
         # Try again, we read_on != None now
         self.accessor.fetch_points(metric, 0, 1, metric.retention[0])
 
+    def test_metric_out_of_search_bounds_should_not_be_found(self):
+        metric_name = "test_metric_out_of_search_bounds_should_not_be_found.a.b.c"
+        metric = self.make_metric(metric_name)
+        self.accessor.create_metric(metric)
+        self.accessor.flush()
+
+        now = datetime.datetime.now()
```

Comment:
Maybe worth looking at metric.created_on instead. It will bring clarity and will allow to test strict boundaries (e.g. equal to the created_on)

Reply:
Done.